docker-entrypoint.sh

```diff
@@ -91,11 +81,7 @@ if [ "$1" = 'postgres' ]; then
 			echo
 		done
 
-		if ! kill -s TERM "$pid" || ! wait "$pid"; then
-			echo >&2 'PostgreSQL init process failed'
-			exit 1
-		fi
-
+		gosu postgres pg_ctl -D ${PGDATA} -m fast -w stop
 		set_listen_addresses '*'
```

Comment:
IMHO, I would keep the function. however, an alternative is to supply the listen_address as option to pg_ctl. That would even render seds obsolote.